media-tv/kodi/kodi-17.0_rc4.ebuild

```diff
@@ -56,7 +56,6 @@ COMMON_DEPEND="${PYTHON_DEPS}
 	>=dev-libs/yajl-2
 	dev-python/pillow[${PYTHON_USEDEP}]
 	dev-libs/libcdio
-	dvd? ( dev-libs/libcdio[-minimal] )
```

Comment:
What does "queued work" mean? The only file change I see is the `rm -rf` below. The VDB change for removing `dev-libs/libcdio` will be triggered by `--new-use`/`-N`